### PR TITLE
updated content

### PR DESCRIPTION
**Pull request by Shashankhs17**

### CODE REVIEW

File: impulse.html [hunk 1]
```diff
@@ -97,7 +95,7 @@ <h4><a href="#hero" style="color: white">IMPULSE</a></h3>
 
         <a href="#hero"><img src="assets/img/logos/impulse.png" alt="Impulse logo" style="width:80%; max-width: 450px; margin-top: 30px;" /></img></a>
         <h2>IMPULSE</h2>
-      <h2>24-Hour Signal Processing Hackathon <br> <h6>National Institute of Technology Karnataka</h6>  <h4 style="color: azure;"> 20th-24th JANUARY 2021 </h4>  </h2>
+      <h2>24-Hour Signal Processing Hackathon <br> <h6>National Institute of Technology Karnataka</h6>  <h4 style="color: azure;"> 11<sup>th</sup> - 15<sup>th</sup> JANUARY 2023 </h4>  </h2>
       <a class="btn btn-primary btn-xl js-scroll-trigger" href="https://forms.gle/7AojpxFt5jr44A7N9">Register Now</a>
```
Comment (nishant-nayak) [inline impulse.html:99]: Update the registration link here.

File: impulse.html [hunk 2]
```diff
@@ -271,138 +257,154 @@ <h5 class="arrow display-5" style="size: .5em;">
...
                         </a>
                       </button>
+
                       <div id="collapse1" class="panel-collapse card-body noScroll collapse" role="tabpanel" data-parent="accordion" aria-labelledby="FAQ-1">
                         <p class="panel-text display-7 panel-body">
-                            Following are eligible to participate - <br>
-                            1) All 1st and 2nd year BTech students of NITK <br>
-                            2) 1st or 2nd year engineering students from any engineering college in India with at least one IEEE member in the team.
-                          </p>
+                          1st or 2nd year engineering students from any engineering college in India are eligible to participate.
```
Comment (nishant-nayak) [inline impulse.html:263]: Mention B.Tech

File: impulse.html [hunk 3]
```diff
@@ -271,138 +257,154 @@ <h5 class="arrow display-5" style="size: .5em;">
...
                           <h5 class="arrow display-5">
-                            The preliminary test is on Hackerearth. Is it a competitive coding test?                          </h5>
+                            The preliminary test is on Hackerearth. Is it a competitive coding test?
+                          </h5>
                         </a>
                       </button>
                       <div id="collapse5" class="panel-collapse card-body noScroll collapse" role="tabpanel" data-parent="accordion" aria-labelledby="FAQ-5">
                         <p class="panel-text display-7 panel-body">
-                            No. The test will have questions about the basics of Linear Algebra, Signals and Systems, Aptitude and some Python coding. It will not have any competitive coding style questions.                       </p>
+                          No. The test will have questions about the basics of Linear Algebra, Signals and Systems, Aptitude and some Python coding. It will not have any competitive coding style questions.
```
Comment (nishant-nayak) [inline impulse.html:312]: Confirm if the prelim test is on Hackerearth or not.

**Review by nishant-nayak — CHANGES_REQUESTED** [review]
- [ ] Update the button on home landing page from Electrika to Impulse
- [ ] Get the updated registration form link from @Jimmy622 and update the register button
- [ ] Update line 116 to say open to only first and second year students from all over India
- [ ] Ensure that the prizes are updated, as well as the certificates for participants

File: impulse.html (hunk 3 above)
Comment (Shashankhs17) [reply impulse.html:320]: Confirmed as test is on Hackerearth itself.

**Comment by Shashankhs17** [comment]
Screenshot (329)
Screenshot (328)

**Comment by Shashankhs17** [comment]
image

**Review by nishant-nayak — APPROVED** [review]
LGTM!

**Pull request merged**